src/main.cpp

```diff
@@ -2451,20 +2498,14 @@ void static UpdateTip(CBlockIndex *pindexNew) {
     static bool fWarned = false;
     if (!IsInitialBlockDownload() && !fWarned)
```

Comment:
I know this is not changed in this PR, but why don't we want these warnings on initial block download?
Would it be a problem to make these checks in ConnectBlock() rather than UpdateTip() ?